src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -378,13 +378,21 @@ bool program_node::has_padded_dependency() const {
     });
 }
 
-std::map<size_t, memory::ptr> program_node::get_const_memory_deps() const {
+std::map<size_t, memory::ptr> program_node::get_const_memory_deps(bool exclude_fused_dep) const {
     std::map<size_t, memory::ptr> mem_deps;
     for (auto& i : get_shape_infer_dependencies()) {
         // Some primitives may have flexible count of deps (e.g. reshape), thus allow skipping some deps
         if (i >= get_dependencies().size())
             continue;
 
+        // exclude fused dependency
+        if (exclude_fused_dep) {
```

Comment:
Do we really need this flag? Can't it always be =true?

Reply:
get_kernel_impl_params() has params->memory_deps = get_const_memory_deps();
I think 'exclude_fused_dep' should be false at this time.

Reply:
Why? fused deps are not supposed to be used via `memory_deps`, so to my this check be removed.

Reply:
I think it can be true always.

Reply:
Deleted the 'if' condition